internal/log/log_test.go

```diff
@@ -92,7 +100,18 @@ func (lt *LogTest) RunAllTests(t *testing.T) {
 			lt.descriptionForTest(methodName),
 			func(t *testing.T) {
 				method(testCaseArgs...)
-				assert.Regexp(t, lt.expectedOutput(methodName), lt.CurrentOutput())
+
+				baseMethodName := strings.Replace(methodName, "ln", "", 1)
+				severityLevelStr := strings.ToUpper(baseMethodName)
+
+				assert.Regexp(
+					t,
+					lt.expectedOutput(
+						methodName,
+						severityLevelStr,
+					),
+					lt.CurrentOutput(),
+				)
```

Comment:
This feels like a little funky to me, but these tests are already kind of out of control after the panic fix, so not a blocker for this PR.  We should def do a proper fixup eventually though 😬

Reply:
fair enough